documentation/docs/20-core-concepts/20-load.md

```diff
@@ -216,6 +216,8 @@ export async function load({ data }) {
 }
 ```
 
+> It's important to not modify `data` to include any non-serializable objects since it will be processed by the **server** `load` after the **universal** `load` returns.
```

Comment:
> "[...] processed by the **server** `load` [...]"

Is not quite right. It is just that the result of the server load gets serialized for hydration, and you can modify this  result (the `data` prop) in the universal load function (which runs right after the server load function and before the serialization of the server load function's result).

Maybe this bug could also be fixed if we serialize the server load function's result before running the universal load function.